src/main/java/cart/domain/CartItem.java

```diff
@@ -9,18 +9,35 @@ public class CartItem {
     private int quantity;
     private final Product product;
     private final Member member;
+    private boolean checked;
 
     public CartItem(Member member, Product product) {
         this.quantity = 1;
         this.member = member;
         this.product = product;
+        this.checked = true;
     }
```

Comment:
```suggestion

public static CartItem of(Member member, Product product) {
		return new CartItem(null, 1, product, member, true);
	}
```

정적 팩토리 메서드를 사용하는 방법도 있겠네요.
개인적으로 필드와 1:1 매핑 되는 경우는 생성자로 만들고 그렇지 않은 경우는 정적팩토리 메서드를 사용하는걸 선호합니다 (이건 취향 차이이니 주드의 기준에 맞게 사용하시면 될 것 같아요!)